tutorials/simulators/1_aer_provider.ipynb

```diff
@@ -117,7 +117,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    print("Circuit unitary:\n", np.asarray(unitary).round(5))
</pre><p>What is the reason for this change? It seems to me that the outcome is the same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
This change is because Aer's `save_unitary` changed its return type from numpy.array to quantum_info.Operator. If we don't include this change, it will raise DeprecationWarning like
```
DeprecationWarning: The return type of saved unitaries has been changed from a `numpy.ndarray` to a `qiskit.quantum_info.Operator` as of qiskit-aer 0.10. Accessing numpy array attributes is deprecated and will result in an error in a future release. To continue using saved result objects as arrays you can explicitly cast them using  `np.asarray(object)`.
  print("Circuit unitary:\n", unitary.round(5))
```